### ISSUE TITLE
Bad Sound AVI Recording.

### ISSUE THREAD

**Issue by Danik2343** [snes9xgit/snes9x#762]
Bad Sound AVI Recording.
Emulator recording bad sound for AVI, with x264vfw and any codecs, BizHawk on x264 record good sound.
What could be the problem?
1. Video, Recorded with Snes9x 1.60 (1.61 sound of AVI bad):https://youtu.be/mLSh5rytWP0
2. Video on BizHawk with good sound record:https://youtu.be/hy5vgfuAuZo

**Comment by Danik2343** [comment]
x264vfw from 2017 build

**Comment by Yave-Yu** [comment]
I have no idea why 1.61's recorded AVI has terrible audio, much worse than 1.52 that also have audio issue.

**Comment by Yave-Yu** [comment]
No... 1.60 has this issue too... also I knew reason: If set playback rate to 32kHz in sound, it records well, don't use 44kHz.

**Comment by OV2** [comment]
Snes9x should switch to 32kHz automatically when recording avi, but this does not currently work if you have automatic input rate activated.
